cnm/plugin/main.go

```diff
@@ -207,7 +207,7 @@ func main() {
 
 	// Relay these incoming signals to OS signal channel.
 	osSignalChannel := make(chan os.Signal, 1)
-	signal.Notify(osSignalChannel, os.Interrupt, os.Kill, syscall.SIGTERM)
+	signal.Notify(osSignalChannel, os.Interrupt, syscall.SIGTERM)
```

Comment:
did we test this change?

Reply:
`os.Kill` cannot be trapped, so including it in the signal list is currently ineffective.